mesheryctl/internal/cli/root/system/system_test.go

```diff
@@ -87,6 +87,21 @@ func TestSystemCmdIntegration(t *testing.T) {
 			ExpectError:     false,
 			TimeoutRequired: 0,
 		},
+		//Restart
+		{
+			Name:            "Restart Meshery",
+			Action:          "restart",
+			Args:            []string{"restart"},
+			ExpectError:     false,
+			TimeoutRequired: 0,
+		},
+		{
+			Name:            "Restart Meshery and skiping update",
+			Action:          "restart",
+			Args:            []string{"restart", "--skip-update"},
+			ExpectError:     false,
+			TimeoutRequired: 0,
+		},
```

Comment:
These should be shifted to appropriate places, like after the `system start` test for both docker and k8s.